newton/solvers/solver_vbd.py

```diff
@@ -1968,6 +1990,7 @@ def __init__(
     ):
         super().__init__(model)
         self.iterations = iterations
+        self.integrate_with_external_rigid_solver = integrate_with_external_rigid_solver
```

Comment:
Can you add a comment to explain what this argument means? Is this argument necessary? It looks like `state_in.body_q` and `state_out.body_q` should be used always, but I don't fully understand how the rigid body coupling works.